setup.py

```diff
@@ -109,6 +111,7 @@
             + test_requirements
             + interactive_requirements
             + lint_requirements,
+            "dask": dask_requirements,
```

Comment:
Would it make sense to add the `dask_requirements` to `dev`?

Reply:
It already is implicitly, since it is part of `test_requirements`.